securedrop_client/gui/widgets.py

```diff
@@ -1083,6 +1098,9 @@ def __init__(self, controller: Controller, source: Source):
         # Add widgets to main layout
         layout.addWidget(self.source_widget)
 
+        # Set click handler
+        # self.clicked.connect(self._on_clicked)
```

Comment:
I will remove this after the first review in case there are other changes I need to make.